projects/standard-rulesets/src/examples/requireExample.ts

```diff
@@ -57,6 +62,8 @@ export const requireRequestExamples = (applies: typeof appliesWhen[number]) =>
 
       if (applies === 'always') {
         requestAssertions.body.requirement((value) => {
+          if (!qualifiedContentType(value.contentType)) return;
```

Comment:
Isn't this better to go into the matches block?

Reply:
I was able to make this change for responses because we have a ResponseBody rule, but not a RequestBody rule. Each request can have multiple content types so I can't do this all in matches -- well I could, but it'd have to go in both places